osbs.spec

```diff
@@ -0,0 +1,100 @@
+%global with_python3 1
+
+%global commit 81b7e9f8228ff972bd84861eefa0eecbc37a2af5
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           osbs
+Version:        0.1
+Release:        1%{?dist}
+
+Summary:        Python module and command line client for OpenShift Build Service
+Group:          Development/Tools
+License:        BSD
+URL:            https://github.com/DBuildService/osbs
+Source0:        https://github.com/DBuildService/osbs/archive/%{commit}/osbs-%{commit}.tar.gz
+
+BuildArch:      noarch
+
+BuildRequires:  python2-devel
+BuildRequires:  python-setuptools
+
+%if 0%{?with_python3}
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+%endif
+
+Requires:       python-pycurl
```

Comment:
This is a tough one. The way osbs is written (you've noticed it probably), you can either choose pycurl or request -- you don't need both. The preferred library being pycurl (since it supports kerberos and transfer chunked encoding natively).
Therefore I would be in favor of requiring pycurl only (and probably comment `import requests` in code). What do you think?